vendors/nuvoton/boards/numaker_iot_m487_wifi/CMakeLists.txt

```diff
@@ -149,6 +150,30 @@ target_sources(
         "${afr_ports_dir}/pkcs11/core_pkcs11_pal.c"
 )
 
+# OTA
+afr_mcu_port(ota)
+target_sources(
+    AFR::ota::mcu_port
+    INTERFACE 
+        "${afr_ports_dir}/ota/aws_ota_pal.c"
+        "${afr_ports_dir}/ota/aws_ota_pal_spim.c"
+)
+target_include_directories(
+    AFR::ota::mcu_port
+    INTERFACE
+            "${AFR_ROOT_DIR}/libraries/abstractions/pkcs11/corePKCS11/source/portable/mbedtls/include"
```

Comment:
@cyliangtw, this path is incorrect for the latest commit of `corePKCS11` library on the master branch which is causing CI build check failures. Can you please make sure to update the submodule in your local workspace and update this path?

Reply:
@aggarw13 ,  yes, this folder was removed in corePKCS11#13224d85 on 2020/12/10. 
Fixed CMakeLists.txt in the new commit 4b9bfde and the new attached log shows all pass. 
[ota_test_v202101.txt](https://github.com/aws/amazon-freertos/files/5901705/ota_test_v202101.txt)